db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -1162,8 +1180,14 @@ class NonBatchedOpsStressTest : public StressTest {
     // Lock the whole range over which we might iterate to ensure it doesn't
     // change under us.
     const int rand_column_family = rand_column_families[0];
-    std::vector<std::unique_ptr<MutexLock>> range_locks =
-        shared->GetLocksForKeyRange(rand_column_family, lb, ub);
+
+    // Parallel read and write to the same key with user timestamp
+    // is not supported, as we might not have the expected value of the key
+    // at some given timestamp
+    if (FLAGS_user_timestamp_size > 0) {
+      std::vector<std::unique_ptr<MutexLock>> range_locks =
+          shared->GetLocksForKeyRange(rand_column_family, lb, ub);
+    }
```

Comment:
Didn't quite understand. What's the difference with `Get()`, which does parallel read and write to the same key? (I think the locking in TestGet() was fully removed.)

Reply:
Seems like we don't verify expected state if user timestamp is used in TestGet() https://github.com/facebook/rocksdb/blob/main/db_stress_tool/no_batched_ops_stress.cc#L515 so I just want to align with that. Updating my comments to better reflect my intention.